qutebrowser/browser/webelem.py

```diff
@@ -50,6 +50,7 @@
     Group.url: '[src], [href]',
     Group.prevnext: 'a, area, button, link, [role=button]',
     Group.focus: '*:focus',
+    Group.inputs: 'input[type=text], input[type=password], textarea',
```

Comment:
Okay - I guess something like `input[type=text], input[type=email], input[type=url], input[type=number], input[type=password], input[type=search]` would work (the same checked later in this file in `_is_editable_input`).